src/controllers/plugins/SolTypes.hpp

```diff
@@ -67,8 +67,8 @@ class ThisPluginState
 /// `std::optional<T>` means nil|LuaEquiv<T> (or zero returns)
 /// A return type that doesn't match returns an error
 template <typename T, typename... Args>
-inline nonstd::expected_lite::expected<T, QString> tryCall(
-    const sol::protected_function &function, Args &&...args)
+inline nonstd::expected_lite::expected<T, QString> tryCall(const auto &function,
```

Comment:
Would be nice to ensure we can only call this with `protected_function` or `main_protected_function`